lib/ansible/modules/storage/glusterfs/gluster_peer.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2015 Nandaja Varma <nvarma@redhat.com>
```

Comment:
Is `2015` right ?

Reply:
Yes module was initially done by her in 2015. Later she left the project. Hope it is ok to keep it.

Reply:
Not a problem :)